### PR TITLE
Fix file extension

### PR DESCRIPTION
**Pull request by timbaev**
Return `nil` if file doesn't have extension

### CODE REVIEW

**Review by calebkleveter — APPROVED** [review]

File: Sources/Core/File.swift [hunk 1]
```diff
@@ -13,7 +13,13 @@ public struct File: Codable {
 
     /// The file extension, if it has one.
     public var ext: String? {
-        return filename.split(separator: ".").last.map(String.init)
+        let splitted = filename.split(separator: ".")
```
Comment (tanner0101) [inline Sources/Core/File.swift]: "to split" is irregular and is still "split" in the past tense. Another name for this could be `parts`.

**Review by tanner0101 — CHANGES_REQUESTED** [review]
LGTM overall, just one small typo. 

We should also make sure to fix this here, for Vapor 4:
https://github.com/vapor/http/blob/master/Sources/HTTPKit/Utilities/File.swift#L16

**Review by tanner0101 — APPROVED** [review]
Thanks!

**Comment by penny-coin** [comment]
Hey @timbaev, you just merged a pull request, have a coin! 

You now have 1 coins.

**Pull request merged**